Database/osirose.sql

```diff
@@ -33,7 +33,7 @@ CREATE TABLE `accounts` (
   `online` tinyint(1) DEFAULT '0',
   `login_count` int(11) DEFAULT '0',
   `lastip` varchar(15) DEFAULT '0.0.0.0',
-  `lasttime` int(11) DEFAULT '0',
+  `lasttime` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
This field is named incorrectly. This is the last time the account logged in. When I was talking about using a timestamp for the sessions I meant actually adding the timestamp to the sessions table.

Reply:
But there's already a timestamp in the session table, so I thought you wanted a timestamp as well for the last time the account logged in